### PR TITLE
Update scaling and performance guide for ephemeral storage

### PR DESCRIPTION
**Pull request by ghost**
This is a follow-up to https://github.com/openshift/openshift-docs/pull/10828.

@aheslin 
http://file.rdu.redhat.com/~tmorriso/071718/scaling/scaling_performance/optimizing_storage.html (the 2nd paragraph only)

http://file.rdu.redhat.com/~tmorriso/071718/scaling/scaling_performance/optimizing_ephemeral_storage.html

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@jhou1 @liangxia Please provide QE review.

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

**Comment by tmorriso-rh** [comment]
@kalexand-rh Do you mind doing the peer review for this PR since it's related to the others you reviewed? Thanks!

File: scaling_performance/optimizing_ephemeral_storage.adoc [hunk 1]
```diff
@@ -11,65 +11,77 @@
...
+system daemons. This partition can be consumed by pods by using EmptyDir
+volumes, container logs, image layers, and container writable layers.
+{product-title} manages shared access and isolation of this
 partition.
 
-- *Runtime* -- This is an optional partition which runtimes can use
-for overlay filesystems.  {product-title} will attempt to identify and
+* Runtime
++
+This is an optional partition that runtimes can use
```
Comment (kalexand-rh) [inline scaling_performance/optimizing_ephemeral_storage.adoc]: I'd reword this to start with "Runtime partitions," but I'm not sure how runtimes use them for overlay file systems. I feel like a few extra words are necessary to explain this relationship.

File: scaling_performance/optimizing_ephemeral_storage.adoc [hunk 2]
```diff
@@ -11,65 +11,77 @@
...
+for overlay file systems. {product-title} attempts to identify and
 provide shared access along with isolation to this partition.
-Container image layers and writable layers are stored here.  If the
-runtime partition exists, the `root` partition will not hold any image
+Container image layers and writable layers are stored here. If the
```
Comment (kalexand-rh) [inline scaling_performance/optimizing_ephemeral_storage.adoc]: maybe s/Container image layers and writable layers are stored here./This partition contains container image layers and writable layers.

File: scaling_performance/optimizing_storage.adoc [hunk 3]
```diff
@@ -14,10 +14,16 @@ toc::[]
 Optimizing storage helps to minimize storage use across all resources. By optimizing storage,
 administrators help ensure that existing storage resources are working in an efficient manner.
 
-This guide primarily focuses on optimizing persistent storage.  Local
-ephemeral storage  _(if the ephemeral-storage tech preview in
-{product-title} 3.10 has been enabled)_, for data utilized during the lifetime of pods, has
-fewer options.
+[NOTE]
+====
+This guide primarily focuses on optimizing persistent storage. Local
```
Comment (kalexand-rh) [inline scaling_performance/optimizing_storage.adoc]: I might say something like
"If you enabled the ephemeral storage technology feature preview in {product-title} 3.10, you can also xref:../install_config/configuring_ephemeral.adoc[configure ephemeral storage], which is temporary storage that pods use during their lifetimes. This feature is disabled by default."
